compose-examples/prysm-only/exit-account.yaml

```diff
@@ -0,0 +1,8 @@
+version: "3.7"
+services:
+  exit-validator:
+    container_name: validator-import-launchpad
+    image: gcr.io/prysmaticlabs/prysm/validator:${PRYSM_DOCKER_TAG}
+    command: accounts voluntary-exit --wallet-dir="/data/wallets" --wallet-password-file="/data/passwords/wallet-password" --public-keys=${PUBKEY} --accept-terms-of-use="true" --beacon-rpc-provider="beacon:4000" --prater
```

Comment:
that's for Prater testnet right? on branch `master` that would be `--pyrmont` instead

Reply:
okey, got it, I've tested it with prater testnet and I think it should also work with pyrmont

Reply:
yes I'm positive this will work for Pyrmont too, but branch `master` is for Pyrmont, so we need this flag instead of `--prater`. I'll adjust it for the merge into branch `prater` for testnet Prater later.